lib/api/skus.js

```diff
@@ -14,12 +17,64 @@ di.annotate(skusRouterFactory, new di.Inject(
         'Http.Services.RestApi',
         '_',
         'Promise',
-        'Protocol.TaskGraphRunner'
+        'Protocol.TaskGraphRunner',
+        'uuid',
+        'Http.Services.SkuPack',
+        'osTmpdir',
+        'rimraf'
     )
 );
 
-function skusRouterFactory (waterline, rest, _, Promise, taskGraphProtocol) {
+function skusRouterFactory (
+    waterline, 
+    rest, 
+    _, 
+    Promise, 
+    taskGraphProtocol, 
+    uuid, 
+    skuPack, 
+    tmp,
+    rimraf
+) {
     var router = express.Router();
+    var rimrafAsync = Promise.promisify(rimraf);
+
+    router.skuPackHandler = function(req,res,skuid) {
```

Comment:
I'm thinking bind the our own function to express Rounter is good or not. Here we bind to router should only because we need to visit it in unittest, otherwise keep it as internal function in skus.js should work fine as well. So whether we need to sacrifice code architecture design for unittest. Add other @RackHD/corecommitters  to comment.